### DISCUSSION TITLE
Can not view the revisions page for a gist

### DISCUSSION THREAD

**Discussion by dstearle** [community/community#28415] [Other Feature Feedback, Questions, & Ideas]
So I have a gist with almost 40 revisions so far which I used to be able to view with no issues, however now whenever I go to view them the page times out and I get the unicorn with the message "This page is taking too long to load". I can still view the gist and make edits with no issue. The gist has about five chunky files (four under a thousand lines but the main one I work on is about 15K) in it so maybe gists are just not able to handle that? The only reason I am using a gist in this particular case is that the project I am working does not have a repo (my clients choice) so this was the next best thing I could think of. Any thoughts on how I could I fix this issue or as to why its happening? If worse comes to worse I dont NEED to view the revisions but it would be helpful.

**Comment by radar-blip** [comment]
Would your client agree to a repo if it was private?
Just a thought.

**Reply by dstearle** [reply]
I mean I could make a repo for just the files that I am currently working on but as for the whole project they do not feel comfortable with having a repo private or otherwise for whatever reason.

**Comment by jfullstackdev** [comment]
your last gists were 3 years ago

![image](https://user-images.githubusercontent.com/47092464/183002644-0a333cd6-77e1-44e6-8a7b-38b9fc99f535.png)

unless it's private,

**Reply by dstearle** [reply]
Yes it is set to private.

**Comment by airtower-luna** [comment]
If it's just the web UI being slow, you could [clone the gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists#cloning-gists) to your machine and view revisions (also edit, commit and push, if you want) like any other Git repository. :smiley_cat:
